### ISSUE TITLE
Hide Confirmation Failed Notifications

### ISSUE THREAD

**Issue by ravensnowy** [winauth/winauth#396]
Hi! Thank you so much for your hard work. 

I was just wondering if there's a way to hide all the "Confirmation Failed" notifications. These notifications would still appear even when you choose the "Auto-Confim (silently)" button. It would be so sleek if the program could just work on the background. Thanks!

Things I've tried: 
1. Set as Auto-confirm (silently).

2. Check "Use System Tray Icon", then "Hide Icon and Notifications" on customize icons. It did not work. For some reason, Windows has no power over these notifications. I'm using Windows 7 by the way.

3. Set it on "Auto-Confirm (NOT silently). Check "Use System Tray Icon", then "Hide Icon and Notifications". It does not work as well. 

Thank you so much!

Raven

Raven

**Comment by winauth** [comment]
Yes, there was another request for this so it's being added. "Silent" will also prevent errors.

**Comment by ravensnowy** [comment]
That would be so great! Thank you so much!

**Comment by TheBilbo** [comment]
> Yes, there was another request for this so it's being added. "Silent" will also prevent errors.
Should I redownload winauth app to see changes? Cause your download page still shows "old" 5.1 version as newest.
